docs/hub/spaces-gpus.md

```diff
@@ -113,6 +113,10 @@ Additional lifecycle control over Spaces with upgraded hardware will soon be pro
 
 Additional information about billing can be found in the [dedicated Hub-wide section](./billing).
 
+### Community GPU Grants 
+
+Do you have an awesome Space but need help covering the GPU hardware upgrade costs? We love helping out those with an innovative Space so please feel free to apply for a community GPU grant and see if yours makes the cut! This application can be found in your Space hardware repo settings in the lower left corner under "sleep time settings".
```

Comment:
Maybe I would change the wording in https://huggingface.co/docs/hub/spaces-overview#hardware-resources to be similar to the one here

Reply:
Hi @osanseviero ! Okay sure thing, I'll open a new PR for this change too!

Reply:
Thanks!